src/chess/board.h

```diff
@@ -43,6 +43,7 @@ class ChessBoard {
   ChessBoard(const std::string& fen) { SetFromFen(fen); }
 
   static const std::string kStartingFen;
```

Comment:
Let's make it `const char*` rather than `std::string`. I think this makes it not need initializer at all (e.g. it won't depend on initialization order). I'm not sure it will work but at least I heard that recommendation to have global variables as `const char*` rather than strings.